macos/library/FLEViewController.m

```diff
@@ -34,33 +34,81 @@
 /**
  * Private interface declaration for FLEViewController.
  */
-@interface FLEViewController () <FLEReshapeListener> {
+@interface FLEViewController () {
   NSOpenGLContext *_resourceContext;
 }
 
 /**
- * Responds to system messages sent to this controller from the Flutter engine.
- * Invoked from a static method provided in the engine configuration, thus must be
- * explicitly declared.
+ * The registered FLEPlugin instances, keyed by the channel they are registered on.
  */
-- (void)handlePlatformMessage:(const FlutterPlatformMessage *)message;
+@property NSMutableDictionary<NSString *, id<FLEPlugin>> *plugins;
+
+/**
+ * A list of additional responders to keyboard events. Keybord events are forwarded to all of them.
+ */
+@property NSMutableOrderedSet<NSResponder *> *additionalKeyResponders;
+
+/**
+ * Initializes plugins used by this view controller.
+ * Called from init.
+ */
+- (void)initPlugins;
+
+/**
+ * Internal implementation of addPlugin:. Separated since it is called during init, so must not
```

Comment:
superfluous period?

Reply:
Nope, "addPlugin:" is the name of the method.